Admin components clean (without sidebar)

hazeltonbw opened the pull request:
# Description

This PR removes the manage users button from the admin panel. This PR also includes changes to the admin panel as discussed in weekly meetings with @huss.

1. Remove Manage Users button from Admin Panel
2. Organize settings into their own categories, and give each a header to make it easy for the user to find the settings they want
3. Change "Default site title" to "Site title"
4. Rename Admin Panel to Admin Settings
5. Update/add translations
6. Small CSS tweaks

Partially Addresses #890

## Type of change

- [ ] Note merging this changes the database configuration.
- [x] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

## Limitations

This PR will not be necessary if PR #1316 (Admin Settings w/ Sidebar change) gets merged as it changes the admin settings page completely.

hazeltonbw committed:
Add and remove translations needed for admin settings dashboard

hazeltonbw committed:
Remove unused sidebar translations, remove reference to managing users in tooltip

hazeltonbw committed:
Organize settings by category and give each a header, make CSS changes

hazeltonbw committed:
Remove unused CSS, change name of admin panel, remove manage users button

hazeltonbw committed:
Fix formatting errors

huss commented on src/client/app/components/admin/PreferencesComponent.tsx:
```diff
@@ -47,18 +47,7 @@ export default function PreferencesComponent() {
-					maxLength={50}
-				/>
-			</div>
+			<h3 className='border-bottom'>Graph Settings</h3>
```
This and the other two section headers are not translated.

Could this entire section be put last on the page? It is the least commonly used so it seems best last (as were many of these items before).

hazeltonbw replied:
Fixed in [e74a45d](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/e74a45dfd49245ab71551a31a40010a1f144186f)

huss commented on src/client/app/components/admin/PreferencesComponent.tsx:
```diff
@@ -312,6 +315,8 @@ export default function PreferencesComponent() {
+
+
 			<Button
 				type='submit'
```
I think it would be nice to have a discard button to undo changes. This may be outside the scope of this PR so it is okay to add this to the linked issue (or a new issue if needed).

hazeltonbw replied:
Made a new issue for this #1326

huss commented on src/client/app/components/admin/PreferencesComponent.tsx:
```diff
@@ -312,6 +315,8 @@ export default function PreferencesComponent() {
 			</div>
+
+
 			<Button
```
I think this would look better visually if there was some space above to separate from the previous item.

hazeltonbw replied:
Fixed in [e74a45d](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/e74a45dfd49245ab71551a31a40010a1f144186f)

huss commented:
This is to note that this PR should not  be merged until PR #1306 is merged. This is due to the fact that the user button is (correctly) removed from this page in anticipation of the new user page/modals.

huss reviewed (changes requested):
Thanks to @hazeltonbw for this contribution. I've made a couple of comments to address but overall this looks good.

hazeltonbw committed:
Made requested changes

Moved graph section to the bottom of the page
Added margin-top to separate settings to make it more readable

huss commented on src/client/app/components/admin/PreferencesComponent.tsx:
```diff
@@ -213,105 +224,99 @@ export default function PreferencesComponent() {
+				</div>
 			</div>
+
+
```
Sorry to just notice this now but on all the modal pages the submit and other buttons are to the right. I think this one should be too for consistency. To work well, I think the entire page width should be limited since it can be made very wide and that only puts the drop down items way to the right. This would also put the buttons in a reasonable place on the right. You can either do this or add to the issue you opened on the discard button since related.

hazeltonbw replied:
Thanks for the comments. I have addressed these changes in commit [93a3575](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/93a3575b4e943ccf0d30376e562c26f8b588dca3). Please let me know what you think, thanks!

huss commented on src/client/app/components/admin/PreferencesComponent.tsx:
```diff
@@ -47,100 +47,111 @@ export default function PreferencesComponent() {
 				failureMessage='failed.to.submit.changes'
 			/>
-			<div style={bottomPaddingStyle}>
+			<h3 className='border-bottom'>{translate('meter.settings')}</h3>
```
I thought I had an earlier comment on this but I cannot find it. Thus, I think either the sections should be alphabetical (best if going with side bar) or it should be graph then site then meter (order likely changed). I'll let you choose what you think is best but I'm leaning toward alphabetical (by English).

hazeltonbw replied:
The setting sections was initially alphabetical, however in your [comment](https://github.com/OpenEnergyDashboard/OED/pull/1324#discussion_r1700553357) you requested a change to move Graph Settings to the bottom. 
I have reverted that change in commit [ec86c72](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/ec86c727c93cd62fcd423ca20dcab27b5cee11fe) and made the setting sections alphabetical.

huss replied:
That was my mistake. I had meant to ask to have the meter settings last (consistent with this comment). However, you chose alphabetical per above and that is fine. I'm hoping we use the sidebar and then this is a good choice.

huss reviewed (changes requested):
Thanks for the update to address the comments. Overall, this looks good. I made a couple of comments that I think should not be hard to address.

hazeltonbw committed:
Change categories back to being alphabetical

hazeltonbw committed:
Limit size of inputs to 50% of the page, 100% when small display is used. Move submit button to the right. Add border to  mimic modal look.

hazeltonbw committed:
Fix formatting errors

huss commented:
Thanks to @hazeltonbw for the quick changes and taking the time to address everything instead of a future issue. This works as expected and is ready to merge.
